Classes/NodeCreationHandler/TemplatingDocumentTitleNodeCreationHandler.php

```diff
@@ -52,8 +53,26 @@ public function handle(NodeInterface $node, array $data)
                 $title = $this->eelEvaluationService->evaluateEelExpression($titleTemplate, $context);
             }
         }
+        
+        $uriPathSegmentTemplate = $node->getNodeType()->getOptions()['template']['properties']['uriPathSegment'] ?? '';
```

Comment:
It's not quite clear why you coerce to `''` and not to `null` here.
It works, but looks a bit weird.

Reply:
I just wanted to write it the same it was done with `$titleTemplate`

Reply:
But for the title it was actually used as a fallback, in your case it is not, right?

Reply:
How you would write it?
```php

Reply:
For instance. Or just

```php
$uriPathSegmentTemplate = $node->getNodeType()->getOptions()['template']['properties']['uriPathSegment'];
if ($uriPathSegmentTemplate) {
```

I'm not a PHP guy so I don't know the right style that should be used here.

Reply:
The above is not allowed because that would throw at least a notice (or a warning?) if the keys were not set, there is no undefined type in PHP so it does that.

Reply:
@kitsunet  So it is correct in the PR?